ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -13,7 +13,7 @@ retry_dnf() {
     local monitor_threshold="$1"
     shift
 
-    local args=("dnf" "-y" "${@}")
+    local args=("dnf" "-y" "--nobest" "${@}")
```

Comment:
Do we really want to make all the dnf commands use `--nobest` or is this better placed here?
https://github.com/daos-stack/daos/blob/d3fcaf474f8ebb5299e54c162c25bf2e088ae025/ci/provisioning/post_provision_config_common_functions.sh#L298
```
            if ! retry_dnf 360 --nobest install $INST_RPMS; then
```